bug_crowd/test.py

```diff
@@ -71,7 +71,7 @@ def test_get_api_uri_for_submission(self):
         """
         submission = get_example_submission()
         expected_uri = self.client.base_uri + (
-            'submissions/%s' % url_quote(submission['uuid']))
+                'submissions/%s' % url_quote(submission['uuid']))
```

Comment:
Also this line ?

Reply:
Looks like my linter added the tab. I'll remove this one too.